frontend/integration-tests/views/catalog.view.ts

```diff
@@ -4,6 +4,11 @@ import { browser, $, $$, by, ExpectedConditions as until, ElementFinder } from '
 
 const moveTo = (elem: ElementFinder) => browser.actions().mouseMove(elem).perform().then(() => elem);
 
+const withRetry = (action) => (args) => action(args)
+  .catch(() => action(args))
+  .catch(() => action(args))
+  .catch(() => action(args));
```

Comment:
Do we know why this is needed? I'd prefer to fix the underlying cause.

Reply:
We get the same error sometimes in the annotation tests, too. I suspect we're iterating over DOM elements that are replaced in the page. Maybe it's the use of `filter` in `entryRowFor`?